src/axom/slic/tests/slic_macros.cpp

```diff
@@ -467,11 +466,19 @@ TEST(slic_macros, test_macros_file_output)
 
   EXPECT_EQ(with_fmt_buffer.str(), "Test");
 
-// Cleanup generated files (not working Windows)
-#ifndef WIN32
+  no_fmt_contents.close();
+  with_fmt_contents.close();
+
+// Closes open file streams associated with Slic streams when destructors
+// called during slic::finalize().
+// Windows _unlink file deletion fails if file is still in use.
+#ifdef WIN32
```

Comment:
Unless there's a reason non-Windows should not finalize(), let's please remove the guard on `WIN32` here and in the slic_macros_parallel.cpp test.  If there is a reason to test removeFile() on non-finaliz()ed slic on non-Windows platforms, I would appreciate knowing.

Reply:
Guard removed! `finalize()` call will work on Windows and non-Windows systems, guard was a relic of my testing.